source/_integrations/tag.markdown

```diff
@@ -42,6 +42,30 @@ Home Assistant has a dedicated panel that allows you to manage your tags. You ca
 
 ![Tag user interface in Home Assistant](/images/blog/2020-09-15-home-assistant-tags/tag-ui.gif)
 
+
+## Using Scanned Tags as a Trigger
+
+To use a tag scan as the trigger for an automation use the ```tag``` platform:
+
+{% raw %}
+
+```yaml
+
+automation:
+  - id: '12000'
+    alias: Lamp NFC
+    initial_state: 'on'
+    trigger:
+      platform: tag
+      tag_id: lamp_nfc_tag
```

Comment:
So where does this `lamp_nfc_tag` come from?

Reply:
I'm not sure I understand. The first half of the document does a great job laying out how to create and manage tags. Do I need to reference the previous paragraphs?

Reply:
I dunno. I'm just looking at it from a user perspective, that sees a `tag_id` here all of a sudden with a value. Where did it come from? Should I replace that or not? Maybe it helps writing an introduction to your example.